scripts/buildsystems/msbuild/vcpkg.targets

```
@@ -226,6 +226,15 @@
     <!-- Search %PATH% for pwsh.exe if it is available. -->
     <Exec
       Condition="'$(VcpkgXUseBuiltInApplocalDeps)' != 'true'"
+      Command="where.exe pwsh.exe &gt;NUL 2&gt;&amp;1"
+      IgnoreExitCode="true"
+      UseCommandProcessor="true">
```

Sorry for the inconvenience. It seems better to set this to `false`.
```suggestion
      UseCommandProcessor="false">
```